### ISSUE TITLE
Add $HOSKY as an asset in the asset list

### ISSUE THREAD

**Issue by victormundi** [Berry-Pool/nami-wallet#181]
Love your wallet! But why is $HOSKY not listed as an asset? I know it's a pile of poo, but still should be in the asset list imo (valued at $0 of course)

**Comment by AngelCastilloB** [comment]
It seems that the $HOSKY team minted the tokens following the [CIP-25 ](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0025) NFT metadata standard:

https://pool.pm/a0028f350aaabe0545fdcb56b039bfb08e4bb4d8c4d7c3c7d481c235.HOSKY

So it is flagged as a non-fungible token.

**Comment by victormundi** [comment]
That is correct. But does that mean that you can't make it show under assets instead of collectibles? Because of the error (?) by $HOSKY team?

**Comment by AngelCastilloB** [comment]
I am not a dev on this project, just adding my 2 cents. But I don't imagine hardcoding this kind of exceptions on your software every time a project incorrectly tags it's token as an NFT it's wise.
